main.py

```
@@ -252,8 +275,12 @@ def build_metadata(item):
 logging.debug("Pushing Quartzy Inventory to eLabFTW...")
 existing_qid_map = {}
 
-response = itemsApi.read_items(_preload_content=False, limit=1500)  # quartzy data has proven to be more than 1300
-items = json.loads(response.data.decode("utf-8"))
+try:
+    response = itemsApi.read_items(_preload_content=False, limit=1500)
+    items = json.loads(response.data.decode("utf-8"))
+except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)